0_intro/exercises/hello_world/src/main.rs

```diff
@@ -1,3 +1,3 @@
 fn main() {
-    todo!("Print hello world on stdout! (not stderr)")
+    println!("Print hello world on stdout! (not stderr)")
```

Comment:
:0 This prints `Print hello world on stdout! (not stderr)` while we just want it to print `hello world`!